drivers/telescope/lx200ap_experimental.cpp

```diff
@@ -591,12 +710,85 @@ bool LX200AstroPhysicsExperimental::ISNewSwitch(const char *dev, const char *nam
 
 bool LX200AstroPhysicsExperimental::ReadScopeStatus()
 {
+    double lng = LocationN[LOCATION_LONGITUDE].value;
+    double lst = get_local_sidereal_time(lng);
+    // 2020-06-02, wildi, isParked is reserved for the state in ParkData.xml
+    // see method isParked()
+    bool isAPParked ;
+    getMountStatus(&isAPParked);
+    if (!isAPParked)
+    {
+        // in case of simulation, the coordinates are set on parking
+        HourangleCoordsNP.s = IPS_BUSY;
+        IDSetNumber(&HourangleCoordsNP, nullptr);
+        HourangleCoordsN[0].value = get_local_hour_angle(lst, currentRA);
+        HourangleCoordsN[1].value = currentDEC;
+        HourangleCoordsNP.s = IPS_OK;
+        IDSetNumber(&HourangleCoordsNP, nullptr );
```

Comment:
why are we setting this twice? Also, maybe we should add a check that in case there is no change from last update, it shouldn't spam the client with unnecessary messages?

Reply:
I agree (see above  "HA is update"). I reconsidered all  HourangleCoordsNP occurrences.